### PR TITLE
Business title

### PR DESCRIPTION
**Pull request by NurkaAmre**
## Iteration - 1

[https://ywgc.atlassian.net/jira/software/projects/YWGC/boards/1?selectedIssue=YWGC-14](url)

## Work done

- [ ]  Business title and description
- [ ] The phone number on the home page
- [ ] Remove the existing navigation bar for now

## Manual testing instructions

- [ ] Resize the screen to a smaller size to simulate a mobile device.
- [ ] Check if the website is responsive and adjusts properly to the smaller screen size.
- [ ] Verify that all elements, including text, images, and buttons, are properly aligned and legible on the smaller screen.

## Gotchas/What I learned

- [ ] The website is responsive and provides a seamless user experience on mobile devices by testing different screen sizes.

### CODE REVIEW

**Review by jaynaxissa — APPROVED** [review]
Looks good!

File: src/pages/Layout.jsx
Comment (novellac) [inline src/pages/Layout.jsx:8]: issue: Same as previous - it's ok to delete! :)

File: src/pages/Layout.jsx
Comment (novellac) [inline src/pages/Layout.jsx:3]: issue: This is going to create more tech debt - you can safely delete this and we'll import it again if we need to.

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:12]: A few things to note here:
issue: Like in the previous comments, since we are using the HTML in a standard way, we don't need to add a role. The `button` element will take care of telling assistive technology that it is a button.
issue: This is a button but doesn't do anything. If you want to use a button, we'll need to handle the click.
issue: The `aria-label` will replace all of the button text on screen readers (SR), so someone using a screen reader would only hear "Phone number," but wouldn't hear what the phone number is. You'll either need to ensure that the phone number is also part of the `aria-label`, or if you have some text on the page you want to associate, you can use [`aria-describedby`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby).
issue: This shouldn't be a button. A good basic rule I try to stick to is that "Buttons do things, Links go places." When you click on the telephone, you are going to navigate away from your website and onto the phone/skype/facetime app. Let's use a link instead of a button. Read a bit more about [linking to phone numbers](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_telephone_numbers) and give it a try.

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:8]:
```suggestion
        <p className="text-[24px] px-[38px] font-body" role="description">
```
issue: We don't need to provide a role here, since we are not using the `p` tag in any special way. Also, `description` isn't a valid role that I know of. I'd remove the `role="description`.

suggestion: It could be helpful to read through this [guide to aria roles](https://www.a11yproject.com/posts/an-indepth-guide-to-aria-roles/), since we may want to use roles later in the project.

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:7]:
```suggestion
        <h1 className="text-[50px] md:text-[64px] text-center font-header font-bold" role="heading">Yorkshire Woods Ground Crew</h1>
```
request: This is not wrong, but I think it's a missed opportunity to componentize our app. Let's create a "BaseHeading" component and use it here.

issue: `role="heading"` is not necessary here. Since we are using an h1, it will tell assistive technology that it's a heading and what level, etc.

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:5]: issue: Please use the Tailwind default sizing (ex: `mt-20`) instead of pixel sizes (ex: `mt-[999px]`). Feel free to push back on design if their measurements aren't in the TW default sizes.

File: tailwind.config.js
Comment (novellac) [inline tailwind.config.js:14]: request(nb): Would you mind taking the fonts work out and doing it in another PR? I think this PR may be a lot, and I don't want to hold up other work.

File: public/index.html
Comment (novellac) [inline public/index.html:24]: request(nb): Would you mind taking the fonts work out and doing it in another PR? I think this PR may be a lot, and I don't want to hold up other work.

**Review by novellac — CHANGES_REQUESTED** [review]
Great start @NurkaAmre! I've left some comments below. I like to use the [conventional comments style](https://conventionalcomments.org/), so that means I've prefixed the comments with the "type" of comment. Any comments with `(nb)` means "non-blocking". I'll approve a PR even if you don't address non-blocking comments.

File: src/pages/Home.jsx
Comment (NurkaAmre) [reply src/pages/Home.jsx:7]: @novellac Sorry I am not getting this point "Let's create a "BaseHeading" component and use it here".
Can you please clarify this?

Comment (novellac) [reply src/pages/Home.jsx:7]: Create a new component file and name it `BaseHeading.jsx`. Inside of the file, add the html tag and the styling. Allow users to pass a prop with the text that you want to put into the base heading component.

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:16]: issue: This is a link so it shouldn't need a click handler. I think you can remove this line.

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:14]: issue: Let's try to keep out any custom hex codes like `bg-[#1B8341]`. Can you ask the designer to choose one of the Tailwind default colours?

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:9]: issue: Remember to use Tailwind default values instead of declaring your own. If this is from the design, feel free to push back with the designer - we've discussed this with the designer and they are okay with that.

File: src/pages/Home.jsx
Comment (novellac) [inline src/pages/Home.jsx:6]: issue: Is there a reason that this `div` has an `aria-label`? I don't think it's necessary, but please do push back to me if you have a reason you'd like to use it.

File: src/components/BaseHeading.jsx
Comment (novellac) [inline src/components/BaseHeading.jsx:8]: issue: Let's use default Tailwind values instead of the bracketed pixel values (ex: `text-[50px] md:text-[64px]`) Please push back with the designer if there doesn't exist a TW value.

File: public/lawnmower.svg
Comment (novellac) [inline public/lawnmower.svg:1]: question(nb): Why are we adding images to this PR? I don't see them being used anywhere. I'll mark this as NB (non-blocking), but in the future, try to make only the changes directly associated with the PR work.

File: public/grabage.svg
Comment (novellac) [inline public/grabage.svg:1]: question(nb): Same question as on the lawnmower logo image file.

File: src/pages/Home.jsx
Comment (NurkaAmre) [reply src/pages/Home.jsx:9]: @novellac I am not getting you. Previous review u mention this

 ` "issue: Please use the Tailwind default sizing (ex: mt-20) instead of pixel sizes (ex: mt-[999px]). Feel free to push back on design if their measurements aren't in the TW default sizes."`

 I am lost.

File: src/pages/Home.jsx
Comment (NurkaAmre) [reply src/pages/Home.jsx:6]: removed ✅

File: src/components/BaseHeading.jsx
Comment (NurkaAmre) [reply src/components/BaseHeading.jsx:8]: Is there a specific reason to use Tailwind values for font sizes? The end result is the same, and it might be more convenient to directly add the px unit in Figma when using Tailwind values.

File: public/grabage.svg
Comment (NurkaAmre) [reply public/grabage.svg:1]: Allie assigned me the task of working on the YWGC service area and designing the corresponding card. Amanda mentioned that it's still in progress and will be ready in the next couple of days. Unfortunately, I had already prepared to download the necessary pictures and other resources, but it turns out that Jayna is already working on it. That's why the gifs are already present.

File: public/lawnmower.svg
Comment (NurkaAmre) [reply public/lawnmower.svg:1]: Allie assigned me the task of working on the YWGC service area and designing the corresponding card. Amanda mentioned that it's still in progress and will be ready in the next couple of days. Unfortunately, I had already prepared to download the necessary pictures and other resources, but it turns out that Jayna is already working on it. That's why the gifs are there.

File: src/components/BaseHeading.jsx
Comment (novellac) [inline src/components/BaseHeading.jsx:8]: issue(nb, this won't impact us just yet 😄 ): Let's make the style dependent on the heading level. We can see in the Figma that `text4xl` and `md:text-6xl` apply to `h1`, but what about when we want to use this component for `h2`, `h3`, etc? Let's make those styles dynamic.

File: tailwind.config.js
Comment (novellac) [inline tailwind.config.js:8]: nitpick(nb, it doesn't hurt anyone): We should lint this so we get standard indentations.

**Review by novellac — APPROVED** [review]
Great job, let's merge your first PR 👶  🎈 🚚 🎉

**Pull request merged**
